src/components/form/widgets/slider.ts

```diff
@@ -32,6 +33,7 @@ export class Slider extends React.Component {
                 step: schema.multipleOf,
                 factor: factor,
                 unit: unit,
+                ...additionalProps,
```

Comment:
I think this, in combination with https://github.com/Lundalogik/lime-webclient/pull/1704/files#diff-1fa34b1dd5ce6645bced6e46655ca008385d2f1784a20b1d265c1a6203bd07a8R79, will overwrite any other classes set on `<limel-slider>`… for example if the component would set any classes on itself via the `Host` element…

Perhaps it's not actually a problem though… it's probably not work making a workaround for at the moment anyway… 🤔

It's internal implementation regardless, so we can just change it later if it turns out to be a problem…